### PR TITLE
Forums notification settings in Member’s Email Preferences to stop receiving email

### LINKED ISSUE #1770: Forums notification settings in Member’s Email Preferences to stop receiving email
**Is your feature request related to a problem? Please describe.**

Added options to stop receiving emails from subscribed topic's reply, 
And someone created a new discussion on subscribed forums.
http://prntscr.com/vinkdi

Below the "SOCIAL GROUPS"
https://prnt.sc/vie2w6

### PR DESCRIPTION
**Pull request by KartikSuthar**
### All Submissions:

* [x] Have you followed the [Pull Requests Guidelines](https://github.com/buddyboss/buddyboss-platform/wiki/Submitting-Pull-Requests#pull-request-guidelines)?
* [x] Does your code follow the [WordPress' Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/)?
* [x] Have you checked to ensure there aren't other open [Pull Requests](../pulls) for the same update/change?

### Changes proposed in this Pull Request:

Fixes #1770 

### Proof Screenshots or Video
http://somup.com/cYXI2KIB1V

### Other information:

* [x] Have you added an explanation of what your changes do and why you'd like us to include them?
* [ ] Have you written new tests for your changes, as applicable?
* [x] Have you successfully run tests with your changes locally?

API PR: https://github.com/buddyboss/buddyboss-platform-api/pull/253

### CODE REVIEW

File: src/bp-forums/groups.php [hunk 1]
```diff
@@ -1487,5 +1490,97 @@ public function map_activity_to_group( $args = array() ) {
 
 			return $args;
 		}
+
+		/**
+		 * Add Forum/Topic Subscribe email settings to the Settings > Notifications page.
+		 *
+		 * @since BuddyBoss 1.5.5
```
Comment (chetansatasiya) [inline src/bp-forums/groups.php:1497]: this should be a `1.5.9`

**Review by chetansatasiya — APPROVED** [review]

**Review by chetansatasiya — APPROVED** [review]

File: src/bp-forums/groups.php [hunk 2]
```diff
@@ -1487,5 +1490,97 @@ public function map_activity_to_group( $args = array() ) {
...
+											class="bp-screen-reader-text"><?php esc_html_e( 'No, do not send email', 'buddyboss' ); ?></span></label>
+							</div>
+						</td>
+					</tr>
+					<?php
+
+					/**
+					 * Fires inside the closing </tbody> tag for forums screen notification settings.
+					 *
+					 * @since BuddyBoss 1.5.5
```
Comment (chetansatasiya) [inline src/bp-forums/groups.php:1576]:
```suggestion
					 * @since BuddyBoss 1.5.9
```

**Review by chetansatasiya — APPROVED** [review]

**Comment by sumitbuddyboss** [comment]
Hello @KartikSuthar 

Fixed >

I have checked Forums notification settings in Member’s Email Preferences to stop receiving email and it's working as expected.


Forum > Email Preferences Setting
A member replies to a discussion you are following.
https://www.loom.com/share/7e1cef6ba8a249339d130a02626efc42

A member has creates a new forum discussion you are following
https://www.loom.com/share/5fb18916b7f04396853a975f35928c38

Note:
@sourabhmatolia 
As we discussed we need to change the text `Following` to `Subscribe`
https://prnt.sc/12x9o5b

Ready for release 🚀 

Thank you

**Comment by KartikSuthar** [comment]
@sourabhmatolia @sumitbuddyboss typo updated: http://prntscr.com/12y7yx1

**Pull request merged**